OpenRA.Mods.Common/Traits/GivesExperience.cs

```diff
@@ -49,6 +52,11 @@ public void Killed(Actor self, AttackInfo e)
 			var killer = e.Attacker.TraitOrDefault<GainsExperience>();
 			if (killer != null)
 				killer.GiveExperience(exp);
+
+			if (info.GivesPlayerExperience)
+			{
+				e.Attacker.Owner.PlayerActor.Trait<PlayerExperience>().GivePlayerExperience(exp / 100);
```

Comment:
I think it might make more sense to have this configurable, with a default value of 0.  This then also covers the `GivesPlayerExperience` parameter.